packages/forms-web-app/src/controllers/appeals/your-appeals.js

```diff
@@ -11,7 +16,12 @@ exports.get = async (req, res) => {
 		let appeals = await apiClient.getUserAppealsById(user.id);
 		if (appeals?.length > 0) {
 			appeals = appeals.map(mapToAppellantDashboardDisplayData);
-			viewContext = { toDoAppeals: appeals, waitingForReviewAppeals: appeals };
+			const toDoAppeals = appeals.filter((appeal) => {
+				return (
+					isEligibilityCompleted(appeal) && !hasDecisionDate(appeal) && hasFutureDueDate(appeal)
```

Comment:
I don't think we need `isEligibilityCompleted`. If an appeal is DRAFT, it goes in `To Do`; so we just need `isDraft` or something?